pkgs/by-name/ca/catppuccin-gtk/package.nix

```diff
@@ -20,73 +19,61 @@ let
   validVariants = [ "latte" "frappe" "macchiato" "mocha" ];
 
   pname = "catppuccin-gtk";
+  version = "1.0.0";
 in
 
 lib.checkListOfEnum "${pname}: theme accent" validAccents accents
 lib.checkListOfEnum "${pname}: color variant" validVariants [variant]
 lib.checkListOfEnum "${pname}: size variant" validSizes [size]
 lib.checkListOfEnum "${pname}: tweaks" validTweaks tweaks
 
-stdenvNoCC.mkDerivation rec {
-  inherit pname;
-  version = "0.7.5";
+stdenvNoCC.mkDerivation {
+  inherit pname version;
 
   src = fetchFromGitHub {
     owner = "catppuccin";
     repo = "gtk";
     rev = "v${version}";
-    hash = "sha256-DIeMdkEjIcPIf/EgE83F5URHY+lR2+hxdc4wSrruFJ8=";
+    hash = "sha256-HaeFl+08CrPaoKDT8eqimtXzSdmdQiQ9Vl4UK3w7C98=";
   };
 
-  nativeBuildInputs = [ gtk3 sassc ];
-
-  patches = [
-    ./colloid-src-git-reset.patch
-  ];
-
-  buildInputs = [
-    gnome-themes-extra
+  nativeBuildInputs = [
+    gtk3
+    sassc
+    git
```

Comment:
Git? There's no .git in source

Reply:
They are applying patches to colloid via `git apply` https://github.com/catppuccin/gtk/blob/4173b70b910bbb3a42ef0e329b3e98d53cef3350/build.py#L465

Reply:
Oh okay, maybe leave a comment there as this is usually seen in some wrong practices

Reply:
Sure i can.